core/lib/stores.ts

```diff
@@ -11,9 +12,17 @@ type StoreValues<Stores extends ReadableAtom[]> = {
   [Index in keyof Stores]: StoreValue<Stores[Index]>
 }
 
-export function readonlyExport<Store extends ReadableAtom>(
-  store: Store
-): ReadableAtom<StoreValue<Store>> {
+/**
+ * use only for manual write store in tests
+ * otherwise don't permit rewrite store at all
+ **/
+export function forceSet<Value>(store: NanoStore<Value>, value: Value): void {
```

Comment:
I think wer can use `ReadableAtom` instead of `Store`/`NanoStore` here. At least it will allow avoid import renaming.